### PR TITLE
STAC browser UI tweaks

### PR DESCRIPTION
**Pull request by nyalldawson**
A couple of small UI tweaks + improvements for working with STAC in the browser

### CODE REVIEW

**Comment by nyalldawson** [comment]
@uclaros 

> How about setting shortcuts for those actions too, like &Refresh, &Details…, Download &Assets…?

I like the idea, but I think we should defer that to a later PR. There's no shortcuts for any browser data item menus right now, and we'd want to ensure consistency across them. And theres quite a lot of changes that will result in 😁

File: src/gui/stac/qgsstacdownloadassetsdialog.cpp [hunk 1]
```diff
@@ -86,6 +95,25 @@ QStringList QgsStacDownloadAssetsDialog::selectedUrls()
...
+{
+  QTreeWidgetItem *item = mTreeWidget->itemAt( p );
+  if ( !item )
+    return;
+
+  mTreeWidget->setCurrentItem( item );
+  mContextMenu->clear();
+
+  const QString url = item->text( 5 );
+  QAction *copyUrlAction = new QAction( tr( "Copy URL" ), this );
```
Comment (uclaros) [inline src/gui/stac/qgsstacdownloadassetsdialog.cpp:108]: It probably doesn't matter much, but shouldn't parent be `mContextMenu` to properly delete old actions on each right click?

Comment (nyalldawson) [inline src/gui/stac/qgsstacdownloadassetsdialog.cpp:108]:
```suggestion
  QAction *copyUrlAction = new QAction( tr( "Copy URL" ), mContextMenu );
```

Comment (nyalldawson) [reply src/gui/stac/qgsstacdownloadassetsdialog.cpp]: Thanks, fixed!

**Review by uclaros — APPROVED** [review]

**Pull request merged**